autogpt/commands/openapi.py

```diff
@@ -0,0 +1,355 @@
+"""Handles any openapi API spec"""
```

Comment:
Design decisions between moving this out of plugins?

Reply:
I couldn't figure out the proper way to get the plugin interface to work as it's needed for this feature of OpenAPI/OpenAI plugin integrations, and I tried, looks like it fits perfectly for the direct commands interface. If you have some ideas on how to get the plugin interface to work that way please advise. Thanks.